docs/pages/backup.md

```diff
@@ -0,0 +1,119 @@
+# Making backups
+
+## User instructions
+
+A disk-objectstore container is fully contained in its root folder. If the container is not being modified, a backup can be made by just making a copy of this folder. The recommended way is to use the `rsync` tool, as the library was designed to be performant with it and make use of the incremental copying capabilities.
+
+However, the preferred way to make a backup, that is also safe while the container is being used (except for when repacking or deleting files), is to use the built-in CLI command:
+
+```console
+$ dostore backup --help
+Usage: dostore backup [OPTIONS] DEST
+
+  Create a backup of the container to destination location DEST, in a
+  subfolder backup_<timestamp>_<randstr> and point a symlink called `last-
+  backup` to it.
+
+  NOTE: This is safe to run while the container is being used.
+
+  NOTE: the symlink `last-backup` is omitted if the filesystem doesn't support
+  it.
+
+  Destination (DEST) can either be a local path, or a remote destination
+  (reachable via ssh). In the latter case, remote destination needs to have
+  the following syntax:
+
+     [<remote_user>@]<remote_host>:<path>
+
+  i.e., contain the remote host name and the remote path, separated by a colon
+  (and optionally the remote user separated by an @ symbol). You can tune SSH
+  parameters using the standard options given by OpenSSH, such as adding
+  configuration options to ~/.ssh/config (e.g. to allow for passwordless login
+  - recommended, since this script might ask multiple times for the password).
+
+  NOTE: 'rsync' and other UNIX-specific commands are called, thus the command
+  will not work on non-UNIX environments.
+
+Options:
+  --keep INTEGER    Number of previous backups to keep in the destination.
+                    (default: 1)
+  --rsync_exe TEXT  Specify the 'rsync' executable, if not in PATH. Used for
+                    both local and remote destinations.
+  --verbosity TEXT  Set verbosity [silent|info|debug], default is 'info'.
+  --help            Show this message and exit.
+
+```
+
+Example usage:
+
+```console
+$ dostore backup /path/to/backup
```

Comment:
I have a question: how does this command actually figure out _which_ container to back up? A system may have multiple containers right?

I just went in the code and the `pass_dostore` is passing the `Container` and this gets the path from the `dostore -p/--path` option, which defaults to the `DOSTORE_PATH` environment variable or otherwise `dostore` in the current working directory. I doubt this is going to work by default for most cases, so I think you should add here how to specify the path.

Reply:
good catch, added `--path` to the example.